src/Avalonia.Base/Input/NavigationDirection.cs

```diff
@@ -1,5 +1,9 @@
 namespace Avalonia.Input
 {
+    // TODO_FOCUS: WinUI doesn't have First,Last,PgUp,PgDown directions
```

Comment:
I don't remember WinUI API, but these navigation directions are useful for lists.
And from what I remember, UWP might implement these key handlers on a control level.

Reply:
WinUI excluded those options in `FocusNavigationDirection`. IMO, these are better left to implement on individual controls where viewport and other necessary information is readily available. That being said, these come from WPF, same as the existing tab navigation logic so they should already work? I've never tested them. I didn't remove them from the enum as I wanted outside opinions.

https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.input.focusnavigationdirection?view=winrt-22621